packages/datadog-plugin-gcloud-pubsub/src/index.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+function createWrapRequest (tracer, config) {
+  return function wrapRequest (request) {
+    return function wrappedRequest (cfg = { reqOpts: {} }, cb) {
+      let topic
+      if (cfg.reqOpts) {
```

Comment:
When is this not set? Is it for methods that are not publishing or subscribing?

Reply:
It's not set when there's bad user input.